internal/tcti/chain_validation.go

```diff
@@ -12,7 +12,7 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
-package scti
+package tcti
```

Comment:
What do you think about just `ct` for this package?
It's clear it's about `static-ct` since that's what this module implements, and it reads very nicely indeed in the callers, e.g. `ct.ParseOIDs`, `ct.NewLog`, etc.